10/ansible.in

```diff
@@ -69,6 +69,7 @@ inspur.ispim
 inspur.sm
 junipernetworks.junos
 kubernetes.core
+kaytus.ksmanage
```

Comment:
Please add `kaytus.ksmanage` before `kubernetes.core`, for alphabetical sorting :) Same in the other files.

Reply:
Oh, no idea why i looked at `y` instead of instead of `a`:) Fixed, PTAL